src/feedback/admin.py

```diff
@@ -16,20 +50,44 @@ class PersonAdmin(admin.ModelAdmin):
 class VeranstaltungAdmin(admin.ModelAdmin):
     fieldsets = [
         ('Stammdaten', {'fields':
-                            ['typ', 'name', 'semester', 'lv_nr', 'grundstudium', 'evaluieren',
+                            ['typ', 'name', 'semester', 'status', 'lv_nr', 'grundstudium', 'evaluieren',
                              'veranstalter', 'link_veranstalter',
                              ]}),
         ('Bestellung', {'fields': ['sprache', 'anzahl', 'verantwortlich', 'ergebnis_empfaenger', 'auswertungstermin',
                                    'freiefrage1', 'freiefrage2', 'kleingruppen', ]}),
     ]
-    list_display = ('typ', 'name', 'semester', 'grundstudium', 'evaluieren', 'anzahl', 'sprache', 'veranstalter_list')
+    list_display = ('typ', 'name', 'semester', 'grundstudium', 'evaluieren', 'anzahl',
+                    'sprache', 'status', 'veranstalter_list')
     list_display_links = ['name']
     list_filter = ('typ', 'semester', 'grundstudium', 'evaluieren', 'sprache')
     search_fields = ['name']
     filter_horizontal = ('veranstalter', 'ergebnis_empfaenger')  # @see http://stackoverflow.com/a/5386871
     readonly_fields = ('link_veranstalter',)
 
 
+class VeranstaltungStatus(Veranstaltung):
+    class Meta:
+        verbose_name = 'Status der Veranstaltung'
+        verbose_name_plural = 'Status der Veranstaltungen'
+        proxy = True
+
+
+class VeranstaltungStatusAdmin(admin.ModelAdmin):
+    list_display = ('lv_nr', 'name', 'status')
+    list_display_links = ['name']
+    list_filter = ('lv_nr', 'name', 'status')
+    search_fields = ['name']
+    actions = [
+        status_angelegt, status_gedruckt, status_versandt, status_boegen_eingegangen, status_boegen_gescannt
+    ]
+
+
+class LogAdmin(admin.ModelAdmin):
```

Comment:
It sould not be possible for the user to manual manipulate a log. So just drop the admin for log